c_tests/test_Image.cpp

```diff
@@ -67,6 +67,23 @@ TEST_CASE_METHOD(CziImageCreator, "test_image_nothrow", "[Image_Cast_Nothrow]")
   REQUIRE_NOTHROW(ImageFactory::getDerived<uint16_t>(img));
 }
 
+class CziRgbImageCreator
```

Comment:
where does this class get used?

Reply:
Thanks you are right it's no longer used in that file.